prow/scripts/cluster-integration/helpers/gke-common-cleaner.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+#Turn off exit-on-error so that next step is executed even if previous one fails.
+set +e
+
+if [ -n "${CLEANUP_CLUSTER}" ]; then
+    shout "Deprovision cluster: \"${CLUSTER_NAME}\""
+    date
+
+    #save disk names while the cluster still exists to remove them later
+    #DISKS=$(kubectl get pvc --all-namespaces -o jsonpath="{.items[*].spec.volumeName}" | xargs -n1 echo)
+    #export DISKS
+
+    #Delete cluster
+    "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/deprovision-gke-cluster.sh"
+
+    #Delete orphaned disks
+    #shout "Delete orphaned PVC disks..."
+    #date
+    #"${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/delete-disks.sh"
+fi
+
+if [ -n "${CLEANUP_GATEWAY_DNS_RECORD}" ]; then
+    #shout "Delete Gateway DNS Record"
+    date
+    #"${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/delete-dns-record.sh --project="${CLOUDSDK_CORE_PROJECT}" --zone="${CLOUDSDK_DNS_ZONE_NAME}" --name="${GATEWAY_DNS_FULL_NAME}" --address="${GATEWAY_IP_ADDRESS}" --dryRun=false
+fi
+
+if [ -n "${CLEANUP_GATEWAY_IP_ADDRESS}" ]; then
+    shout "Release Gateway IP Address"
+    date
+    "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/release-ip-address.sh --project="${CLOUDSDK_CORE_PROJECT}" --ipname="${GATEWAY_IP_ADDRESS_NAME}" --region="${CLOUDSDK_COMPUTE_REGION}" --dryRun=false
+fi
+
+
+if [ -n "${CLEANUP_DOCKER_IMAGE}" ]; then
+    shout "Delete temporary Compass-Installer Docker image"
+    date
+    KYMA_INSTALLER_IMAGE="${COMPASS_INSTALLER_IMAGE}" "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/delete-image.sh"
```

Comment:
${KYMA_INSTALLER_IMAGE} should be set in calling script. COMPASS_INSTALLER_IMAGE is specific for compass jobs. Others use different variables.

Reply:
I've added both lines and now we are testing if variable is defined, so w only do it when needed